### PR TITLE
Bug fix.

### PR DESCRIPTION
**Pull request by rizard**
Removed some more unnecessary comments. Don't know how hard_timeout and idle_timeout got reverted in the SFP...

### CODE REVIEW

File: src/main/java/net/floodlightcontroller/staticflowentry/StaticFlowEntryPusher.java [hunk 1]
```diff
@@ -365,19 +365,21 @@ void parseRow(Map<String, Object> row, Map<String, Map<String, OFFlowMod>> entri
 				if (row.get(key) == null) {
 					continue;
 				}
+				
 				if (key.equals(COLUMN_SWITCH) || key.equals(COLUMN_NAME) || key.equals("id")) {
 					continue; // already handled
 				}
-				// explicitly ignore timeouts and wildcards
-				if (key.equals(COLUMN_HARD_TIMEOUT) || key.equals(COLUMN_IDLE_TIMEOUT)) {
```
Comment (Sovietaced) [inline src/main/java/net/floodlightcontroller/staticflowentry/StaticFlowEntryPusher.java]: So the notion of the static flow pusher is that it pushes static STATIC flows. If you respect the hard/idle timeout values flows can expire, but will still be displayed by the SFP database which is inconsistent. Make sense? @rizard

Comment (rizard) [inline src/main/java/net/floodlightcontroller/staticflowentry/StaticFlowEntryPusher.java]: Yeah, that makes sense. We wanted to include the ability to do this though.
Many have asked for it in the past, so I figured I'd at least make it an
option. I had intended to do that for the release, but somehow it got
reverted when I did the merge and I didn't realize it. There will be a note
in the SFP documentation that this feature may not be stable.

Ryan Izard
Graduate Research/Teaching Assistant
306B Fluor Daniel Building
ECE Department, Clemson University
Clemson, SC 29634
rizard@g.clemson.edu
---------------------------------------------------
Big Switch Networks
ryan.izard@bigswitch.com

**Comment by rizard** [comment]
@Sovietaced Do you think this would work: set the SEND_FLOW_REM flag in all SFP flows and have the SFP listen for the messages and remove all timed-out flows from its DB?

**Comment by rizard** [comment]
@Sovietaced It actually works out nicely. Reacting to the flow removed message and removing the flow from storage does the trick. The unfortunate side effect though is that a flow delete message is needlessly sent to the switch when the SFP reacts to the row being removed from the table. The switch discards the message silently though, which is a plus.

The assumption made was that the only way a row can be removed is if the removal is initiated by the controller. With a row being removed by the switch due to a timeout, there needs to be another mechanism to convey to the storage listener in the SFP that the removal was due to a timeout.

**Pull request merged**